backend/spec/rails_helper.rb

```diff
@@ -64,4 +64,15 @@
   # config.filter_gems_from_backtrace("gem name")
   config.include JsonResponseHelper
   config.include Devise::Test::IntegrationHelpers
+
+  # These two blocks check for scenarios with bullet: :skip, and if they find
+  # one before they run tests, they will set Bullet.enable to false and then
+  # set it back to true after.
+  config.before(:each, bullet: :skip) do
```

Comment:
I am not sure if we should use this, 
I believe using this skip for bullet can hide some issues that we might have in prod like "lazy load", "eager load" or "n+1".

I can take a look at this so we can identify if it is really necessary or not, I can look at this tonight or maybe earlier, I will ping you